src/components/Main/Honey/style.ts

```diff
@@ -11,6 +11,31 @@ export const HoneyContainer = styled.div`
   width: 100%;
 `;
 
+export const leftSideFloatingNavWrapper = styled.div`
```

Comment:
```suggestion
export const LeftSideFloatingNavWrapper = styled.div`
```
컴포넌트 이기 때문에 대문자로 시작해야 될 것 같아요

Reply:
확인했습니다. 수정하겠습니다.